src/Sheet/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import SheetContainer from './SheetContainer';
+
+export const POSITIONS = {
+  left: 'left',
+  right: 'right',
+  top: 'top',
+  bottom: 'bottom',
+};
+
+export const VARIANTS = {
+  light: 'light',
+  dark: 'dark',
+};
+
+class Sheet extends React.Component {
+  constructor(props) {
+    super(props);
+    this.blockClicks = this.blockClicks.bind(this);
+  }
+
+  blockClicks(event) {
+    event.stopPropagation();
+  }
```

Comment:
The sheet doesn't currently lock focus inside the sheet if it's set to blocking. I suggest using this library: https://github.com/theKashey/react-focus-on to do this work. It does a great job of handling scrolling, clicking, and focus blocking very well.